Formula/latexml.rb

```diff
@@ -26,6 +118,7 @@ def install
     ENV.prepend_create_path "PERL5LIB", libexec+"lib/perl5"
     resources.each do |r|
       r.stage do
+        ENV["PERL_CANARY_STABILITY_NOPROMPT"] = "1"
```

Comment:
```suggestion
        ENV["PERL_CANARY_STABILITY_NOPROMPT"] = "1" unless OS.mac?
```
I tend to add `unless OS.mac?` so that I can easily see which lines are specific to Linux.